api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -611,6 +611,40 @@ type FelixConfigurationSpec struct {
 	// [Default: Auto]
 	BPFConntrackCleanupMode *BPFConntrackMode `json:"bpfConntrackMode,omitempty" validate:"omitempty,oneof=Auto Userspace BPFProgram"`
 
+	// BPFConntrackTimers overides the default values for the specified conntrack timer if
+	// set. It is a key-value make, where each value can be either a duration or `auto` to
+	// pick the value from a Linux conntrack timeout.
+	//
+	// Possible values for the keys are: CreationGracePeriod, TCPPreEstablished,
+	// TCPEstablished, TCPFinsSeen, TCPResetSeen, UDPLastSeen, GenericIPLastSeen,
+	// ICMPLastSeen.
+	//
+	// Unset or incorrect values are replaced by the default values with a warning log for
```

Comment:
What's an incorrect value? Is there some value that can byapss our validation?

Reply:
Old comment, not valid anymore :+1: